docs/cugraph/source/installation/source_build.md

```diff
@@ -1,53 +1,46 @@
 # Building from Source
 
-The following instructions are for users wishing to build cuGraph from source code.  These instructions are tested on supported distributions of Linux, CUDA, and Python - See [RAPIDS Getting Started](https://rapids.ai/start.html) for list of supported environments.  Other operating systems _might be_ compatible, but are not currently tested.
-
-The cuGraph package include both a C/C++ CUDA portion and a python portion.  Both libraries need to be installed in order for cuGraph to operate correctly.
+These instructions are tested on supported distributions of Linux, CUDA, and
+Python - See [RAPIDS Getting Started](https://rapids.ai/start.html) for list of
+supported environments.  Other operating systems _might be_ compatible, but are
+not currently tested.
 
 ## Prerequisites
 
 __Compiler:__
 * `gcc`           version 9.3+
 * `nvcc`          version 11.0+
-* `cmake`         version 3.20.1+
```

Comment:
Is there no longer a cmake version requirement? 
I get cmake version errors frequently.

Reply:
There is still a `cmake` dependency, but I changed this doc so that the "prerequisites" section only mentions things not typically installed by our conda dev environment. Since `cmake` is included in the dev conda environment, I just removed it here for consistency (otherwise it felt like I should also mention make, ninja, cython, etc., etc.)